src/inventory.cpp

```diff
@@ -271,6 +286,17 @@ std::string ItemStack::getShortDescription(const IItemDefManager *itemdef) const
 	return desc;
 }
 
+f32 ItemStack::getRange(const IItemDefManager *itemdef) const
+{
+    std::string str_meta_range = metadata.getString("range");
```

Comment:
This should use a reference, probably a const reference.

Reply:
As I'm new to C++, I'm not exactly sure what this means. What value are you referring to that should be a const? It seems that they are all const. The function is a const and so is the IItemDefManager.

Reply:
I'm sorry, I was unclear. I'm referring to str_meta_range; copying strings is expensive.
```suggestion
    const std::string &str_meta_range = metadata.getString("range");
```